rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -189,6 +190,11 @@
      */
     private final static Logger logger = Logger.getLogger(RMCore.class);
 
+    /**
+     * Log4J logger name dedicated to nodes recovery
+     */
+    private final static Logger nodesRecoveryLogger = Logger.getLogger(RMCore.class + " Nodes Recovery");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)